pkg/controller/argocd/util.go

```diff
@@ -581,7 +563,7 @@ func (r *ReconcileArgoCD) reconcileResources(cr *argoprojv1a1.ArgoCD) error {
 		return err
 	}
 
-	log.Info("reconciling deployments")
+	log.Info("reconciling `deployment`s")
```

Comment:
I don't think this one is quite right.

Reply:
Fixed.  I honestly don't know how that change was made.